README.md

```diff
@@ -1,94 +1,78 @@
+***
 
-[Here is a video](https://youtu.be/WVbxFIGBh0Y) showing the demo project running in the ios simulator.
+### *Android*
 
-[And here](https://youtu.be/_ZzBA28QO4s) is the android version.
+[![Youtube - Android Demo](androidscreenshot.png)](https://youtu.be/_ZzBA28QO4s "Youtube -Android Demo Movie")
 
-Each view:
-===
+### *iOS*
+
+[![Youtube - iOS Demo](iosscreenshot.png)](https://youtu.be/WVbxFIGBh0Y "Youtube -iOS Demo Movie")
+
+## Each View:
+***
 
 - has it's own global context
 - has it's own collection of plugins (instances, classes are shared, so watch for statics )
 - can communicate via localStorage ( assuming they are loaded from the same location )
 -has access to all Cordova resources
 
-Currently Supported Platforms:
-===
+## Currently Supported Platforms:
+***
 
 - iOS
 - Android
 
-This project includes a brief demo project, to run it in ios/Android:
-===
-
-    cd demo
-    cordova platform add ios (or) cordova platform add android
-    cordova plugin add --link .. (Android does not link)
-    cordova run ios (or) cordova run android
-
-The current API supports two views but will eventually have support for multiple views in a stack.
+## This Repo Includes a Brief Demo Project, to Run it in iOS/Android:
+***
 
- Quickstart Guide to using the MultiView plugin on the Android Platform
- ===
+```bash
+$ cd demo
+$ cordova platform add iOS (or) cordova platform add android
+$ cordova plugin add --link .. (Android does not link)
+$ cordova run iOS (or) cordova run android
+```
 
- - To *launch* a new webview make this call in your application's JS:
-  PGMultiView.loadView(url, message, success, error);
-
-      @param 'url' the location of your html file.  The multiview plugin automatically adds the
-      prefix of :///android_asset/www/ if your url does not start with js
-      @param  'message', current version stores data in the form of string.  In the next release 'message' will require JSON objects.
-      @param 'success', a sucessful plugin callback
-      @param 'error', the plugin was not called
-
-      -please note that you must make two separate JS files which correspond respectively to the native portions of the plugin
-      (PGMultiview.java and PGMultiviewActivity.java) if you want to utilize both views.
-
- - To *dismiss* a webview make this call in your application's JS:
-      PGMultiView.dismissView(data);
-      @param 'data' is currently a string - will be JSON object in next release.
+The Demo application illustrates the passing of data between a parent-view and child-view, however the API provides support for 
+*multiple* views in a stack.  Each additional view must have it's own JavaScript file that corresponds with native portions of the plugin.
 
- - Passing Data
+## Installation Instructions
+ ***
+Please download the latest version date version of our plugin directly from NPM:
+[NPM PhoneGap-Plugin-MultiView](https://www.npmjs.com/package/phonegap-plugin-multiview)
```

Comment:
They either fork the repo, and git clone. Or they install via npm.
Install via npm is just `cordova plugin add phonegap-plugin-multiview`